_templates/header.php

```diff
@@ -100,6 +100,7 @@
         <script>
             <?php include __DIR__.'/../scripts/jql.min.js'; ?>
             jQl.loadjQ('https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.0/jquery.min.js')
+            jQl.loadjQdep('https://cdnjs.cloudflare.com/ajax/libs/flipclock/0.7.8/flipclock.min.js')
```

Comment:
I was having some trouble with it being called in the countdown.

- If it's after the `boot` call then it won't work as intended
- If the `boot` call is put in the footer then there is a chance the other scripts will load and jQl won't intercept the `ready` calls
- If it's a regular (non async) call then it won't always load after jQuery is loaded and give an undefined error